distributed/client.py

```diff
@@ -954,8 +954,9 @@ def __init__(
         else:
             self.connection_args = self.security.get_connection_args("client")
 
+        self._loop = loop
         self._asynchronous = asynchronous
-        self._loop_runner = LoopRunner(loop=loop, asynchronous=asynchronous)
+        self._loop_runner = LoopRunner(loop=self._loop, asynchronous=asynchronous)
```

Comment:
You should not set `loop` as an attribute but rather use `Client._loop_runner.loop` if it is needed.

Reply:
are you actually using this?

Reply:
Oh ya, left over from previous iteration, not needed and removed now. https://github.com/dask/distributed/pull/7871/commits/5072d793ff6381d6125550e1bfddeeee0f86d203